### ISSUE TITLE
Report portal connection error appears, when REPORT_PORTAL_BASE_URL=http://localhost:8080

### ISSUE THREAD

**Issue by iegor-zabielin** [redfox256/testcafe-reporter-reportportal#4]
1. Spin up Report Portal locally.
2. Create .env file, specify REPORT_PORTAL_BASE_URL=http://localhost:8080
3. **testcafe v.1.1.2.** (also tried on 0.23.3).
4. Trigger any test execution.

**Actual result:** Error is logged
Error connecting to ReportPortal, confirm that your details are correct.
Error: Request failed with status code 404: {"timestamp":1555152511909,"status":404,"error":"Not Found","message":"No message available","path":"/api/v1/fnn_test/user"}
    at axios.then.catch (/Users/izabielin/Documents/GitHub/fnn-portal/node_modules/reportportal-client/lib/rest.js:27:23)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)

**Even with such error, test results are added on the Report Portal.**

**Comment by redfox256** [comment]
Unable to replicate with the new fix `v1.0.18` - please test

**Comment by iegor-zabielin** [comment]
Unfortunately it is reproduced 
<img width="1381" alt="Screenshot at Apr 18 15-16-05" src="https://user-images.githubusercontent.com/32905475/56360382-23f65380-61ed-11e9-8b87-6a651b9f5969.png">

**Comment by redfox256** [comment]
There seems to be an issue with the [reportportal-client](https://github.com/reportportal/client-javascript/issues/30) being used. I will keep monitoring till fixed.

**Comment by kerly** [comment]
I am noticing this same error when attempting to configure with an externally hosted reportportal (not localhost).

**Comment by iegor-zabielin** [comment]
@kerly yes, it is reproduced not only on localhost...unfortunately

**Comment by ampc** [comment]
Also happening here :+1: 

```
 testcafe chrome tests/specs/login/login.spec.ts --reporter reportportal                                                                                                                          
Using locally installed version of TestCafe.
Running tests in: Chrome 74.0.3729 / Linux 0.0.0

[Login]
Error connecting to ReportPortal, confirm that your details are correct.
Error: Request failed with status code 404: {"timestamp":1558519345518,"status":404,"error":"Not Found","message":"No message available","path":"/api/v1/superadmin_personal/user"}
    at axios.then.catch (/home/ampc/repos/automated_tests/QA_Tests/e2e_smart/node_modules/reportportal-js-client/lib/rest.js:27:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  [ ✓ ] R-1001 | Login with correct credentials
  [ ✖ ] R-1002 | Login with wrong credentials
```

**Comment by aadhithlmi** [comment]
Unfortunately I am also getting the same error 

`PS C:\Code\testcafe> testcafe chrome test01.js --reporter reportportal
Using locally installed version of TestCafe.
Running tests in: Chrome 74.0.3729 / Windows 10.0.0

[Getting Started]
Error connecting to ReportPortal, confirm that your details are correct.
Error: Request failed with status code 404: {"timestamp":1559081127097,"status":404,"error":"Not Found","message":"No message available","path":"/api/v1/axdynamics/user"}
    at axios.then.catch (C:\Code\testcafe\node_modules\reportportal-js-client\lib\rest.js:27:23)
    at process._tickCallback (internal/process/next_tick.js:68:7)
  [ ✓ ] My first test`

**Comment by kerly** [comment]
I updated to the newest version of this package (1.0.22) and it looks like this is fixed now. Can others confirm?

**Comment by redfox256** [comment]
I've updated the reportportal-js-client library to the latest version `v1.0.2` which mentions connection issue fixes.
Please can you confirm that it's fixed in the current version released `v1.0.23`

**Comment by kerly** [comment]
I tried `v1.0.23` and I'm not seeing the reported error anymore.

**Comment by iegor-zabielin** [comment]
@kerly, @redfox256 confirming issus is fixed in `v1.0.23`
